src/modules/UI/components/ControlPanel/Component/Main.js

```diff
@@ -306,15 +304,12 @@ const LogoutButton = ({ onPress }) => {
   )
 }
 
-const goToFioAddressesScene = async (fioAddresses: FioAddress[] | null) => {
-  if (!fioAddresses || !fioAddresses.length) return Actions[Constants.FIO_ADDRESS_REGISTER]()
-  return Actions[Constants.FIO_ADDRESS_LIST]()
-}
-const FioAddressButton = ({ fioAddresses }) => {
+const goToFioAddressesScene = () => Actions[Constants.FIO_ADDRESS_LIST]()
```

Comment:
Unnecessary function goToFioAddressesScene. Just pass `Actions[Constants.FIO_ADDRESS_LIST]` to the onPress prop

Reply:
done